src/create-extension-pack.js

```diff
@@ -0,0 +1,176 @@
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+
+/*
+ * Creates an extension package referencing all built-in extensions previously
+ * created during build time i.e. by executing yarn
+ * 
+ * Extensions will be skipped if a corresponding .vsix file is not found under the 'dist'
+ * folder and also not found under the extension registry.
+ */
+// @ts-check
+const fs = require('fs-extra');
+const path = require('path');
+const vsce = require('vsce');
+const yargs = require('yargs');
+
+const { computeVersion, resolveVscodeVersion, isPublished } = require('./version');
+const { dist, extensions, theiaExtension } = require('./paths.js');
+
+const { tag, force } = yargs.option('tag', {
+    choices: ['latest', 'next']
+}).demandOption('tag')
+    .option('force', {
+        description: 'Create extension pack even if it is found to be already available',
+        boolean: true,
+        default: false
+    }).argv;
+
+const packageJson = 'package.json'
+const categories = ['Extension Packs'];
+const packName = 'builtin-extension-pack';
+const publisher = 'eclipse-theia';
+const repository = 'https://github.com/eclipse-theia/vscode-builtin-extensions';
+
+(async () => {
+    const vscodeVersion = await resolveVscodeVersion();
+    const packVersion = await computeVersion(tag);
+    const extPackNameAndVersion = packName + '-' + packVersion;
+
+    const extPackVsixPath = dist(packName + '-' + packVersion + '.vsix');
+    const extensionPackAlreadyAvailable = await isAvailable(extPackVsixPath, packName, packVersion, publisher);
+    if (extensionPackAlreadyAvailable && !force) {
+        console.log("Exiting as this extension package is already created or published: " + extPackVsixPath);
+        return;
+    }
+
+    const extPackSrcFolder = theiaExtension(extPackNameAndVersion);
+    if (!fs.existsSync(extPackSrcFolder)) {
+        await fs.mkdir(extPackSrcFolder);
+    }
+
+    const extPack = {};
+    extPack.name = packName;
+    extPack.displayName = packName;
+    extPack.description = 'Builtin extension pack associated to a version of vscode';
+    extPack.version = packVersion;
+    extPack.publisher = publisher;
+    extPack.categories = categories;
+    extPack.engines = { vscode: '^' + vscodeVersion };
+    extPack.repository = repository;
+    extPack.extensionPack = await resolveExtensions();
```

Comment:
I believe we are missing the `license` field in the actual `package.json`:

```json
{
  "license": "EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0",
}
```

or an entry like the following:

```json
{
  "license" : "SEE LICENSE IN <filename>"
}
```